.travis.yml

```diff
@@ -55,6 +55,12 @@ jobs:
         - make .gitvalidation
         - make binaries
       go: tip
+    - script:
+        - make install.tools
+        - make .gitvalidation
+        - make binaries
+        - make verify
+      go: 1.8.x
```

Comment:
1.8.x  ?

Reply:
we were having timeouts with lint on 1.9 on travis... so verify was moved to just the pull stage. So here I'm just using 1.8 for early detection of lint and boilerplate errors.